release/release_logs/1.13.0/microbenchmark.json

```diff
@@ -0,0 +1,274 @@
+{
+    "1_1_actor_calls_async": [
+        6303.495416674034,
+        127.61857475096934
+    ],
+    "1_1_actor_calls_concurrent": [
+        4643.127624106235,
+        26.15691889855685
+    ],
+    "1_1_actor_calls_sync": [
+        2292.0521240554867,
+        49.67730630463093
+    ],
+    "1_1_async_actor_calls_async": [
+        3520.7866511820234,
+        144.66711445570607
+    ],
+    "1_1_async_actor_calls_sync": [
+        1540.9621110632384,
+        26.852382548336788
+    ],
+    "1_1_async_actor_calls_with_args_async": [
+        2482.7467673102465,
+        235.92931263724392
+    ],
+    "1_n_actor_calls_async": [
+        11956.002041587353,
+        135.79929253713422
+    ],
+    "1_n_async_actor_calls_async": [
+        11243.062194952357,
+        317.49092121968664
+    ],
+    "client__1_1_actor_calls_async": [
+        951.4342830352992,
+        9.757190101655269
+    ],
+    "client__1_1_actor_calls_concurrent": [
+        943.817403100398,
+        7.482045631696154
+    ],
+    "client__1_1_actor_calls_sync": [
+        546.4884066643845,
+        2.76006293195622
+    ],
+    "client__get_calls": [
+        1176.4758507805825,
+        7.713785640146038
+    ],
+    "client__put_calls": [
+        839.8721108084437,
+        34.66588611022296
+    ],
+    "client__put_gigabytes": [
+        0.0453433760564245,
+        0.00036863456775825215
+    ],
+    "client__tasks_and_get_batch": [
+        0.8853716083224373,
+        0.020227375558576004
+    ],
+    "client__tasks_and_put_batch": [
+        9109.735067057867,
+        89.08464775818196
+    ],
+    "multi_client_put_calls_Plasma_Store": [
+        9662.197103077946,
+        184.95923834781362
+    ],
+    "multi_client_put_gigabytes": [
+        40.94456764236526,
+        1.4646681109150272
+    ],
+    "multi_client_tasks_async": [
+        33373.49615641964,
```

Comment:
this has a huge regression 46643 ->  33373. cc @scv119 do you know anything about this?